cmd/neofs-node/attributes.go

```diff
@@ -11,4 +15,52 @@ func parseAttributes(c *cfg) {
 	}
 
 	fatalOnErr(attributes.ReadNodeAttributes(&c.cfgNodeInfo.localInfo, nodeconfig.Attributes(c.appCfg)))
+
+	// expand UN/LOCODE attribute if any found; keep user's attributes
+	// if any conflicts appear
+
+	locAttr := c.cfgNodeInfo.localInfo.LOCODE()
+	if locAttr == "" {
+		return
+	}
+
+	key, record, err := getRecord(locAttr)
+	if err != nil {
+		fatalOnErr(fmt.Errorf("could not get locode record from DB: %w", err))
+	}
+
+	n := &c.cfgNodeInfo.localInfo
+
+	setLocodeAttr(n, "CountryCode", key.CountryCode().String())
```

Comment:
The way I saw it in #2612 is:
 * `./neofs-cli util locode get-attributes "BE RUS"`
 * copy-paste into config and run

But this code auto-expands one attribute into multiple attributes (and requires some memory on the SN side!).

Now I'm not sure which way is better. Having all parameters listed explicitly in the config with no auto-magic seems to be a bit more robust and predictable, but have some magic already with version and capacity, so maybe it's still OK if documented properly. @532910?

Reply:
> requires some memory on the SN side

agree. that is why i also thought about cleaning memory in the locode lib (a separate call that makes it unusable)

> Now I'm not sure which way is better.

me neither. but i am sure that it is more convenient for a user (taking into account that some attrs are already doing such a magic expansion)

> Having all parameters listed explicitly in the config with no auto-magic seems to be a bit more robust and predictable

and an administrator can do it himself, that is why user's value are prioritized in this PR

Reply:
> cleaning memory in the locode lib (a separate call that makes it unusable)

I had this thought as well. And we can also use an mmaped tmp file for unpacked data (that can be swapped out).

Reply:
After additional discussions, let's leave it as is:
 * with nspcc-dev/locode-db#28 it's just 4M overhead and we manage at least of number of 64M objects
 * it makes configuration 100% compatible
 * it's a bit easier to manage
 * it's friendly to configuration reloads (`Close()` is not)